tests/e2e/scripts/hermes_bootstrap.sh

```diff
@@ -38,11 +38,11 @@ id = '$OSMO_A_E2E_CHAIN_ID'
 rpc_addr = 'http://$OSMO_A_E2E_VAL_HOST:26657'
 grpc_addr = 'http://$OSMO_A_E2E_VAL_HOST:9090'
 websocket_addr = 'ws://$OSMO_A_E2E_VAL_HOST:26657/websocket'
-rpc_timeout = '10s'
```

Comment:
oh rip

Reply:
Lets make a github issue for this

Reply:
I don't think I am on the same page, what issue should I create with regards to this?